scripts/stall-analyser.py

```diff
@@ -158,6 +158,14 @@ def _print(l:str, width:int):
 
     def print_graph(self, direction:str):
         top_down = (direction == 'top-down')
+        print(f"""
+This graph is printed in {direction} order, where {'callers' if top_down else 'callees'} are printed first.
```

Comment:
Idea: this would be a great place to add a message like "(use --some-option to show in {opposite direction} order)" - so users who see this is not the order they expect have an immediate hint on how to reverse the order, instead of needing to check the documentation whether there is even a way to reverse the order, and how.

Reply:
I can do that, but doesn't it hold true for other options as well?
I don't want to cram too much information into the output's preamble.

Reply:
Not all options print a separate log message saying what their chosen value is. I suggest that if you chose to do that, you should say not what the chosen value is - but also which option chose it.

Reply:
@nyh I think that `f"Command line options used: {args}\n"` printout is enough for documenting what were the options in use.
But I'll add the instruction you asked for per your request if you think it's useful.

Reply:
Pushed in 164f2d7c9